tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTest.java

```diff
@@ -941,6 +942,16 @@ public void testNonPublicSymbolsWarningInServiceClass() {
         Assert.assertEquals(result.getDiagnostics().length, 0);
     }
 
+    @Test
+    public void testClassWithModuleDefaultValue() {
```

Comment:
What does module default value mean? 

```suggestion
    public void testClassWithModuleLevelVarAsDefaultValue() {
```

Reply:
Updated. I am trying to specify this test case using a module variable as a default value.